src/ui/server/doc/openapi_v1.json

```diff
@@ -57,6 +57,31 @@
         }
       }
     },
+    "/database/status": {
+      "get": {
+        "operationId": "getDatabaseStatus",
+        "tags": [
+          "Util"
+        ],
+        "summary": "Retrieve information about database status",
+        "description": "Returns current legal directive of the data as well as the date of the last update of the database.",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/DatabaseStatus"
```

Comment:
Please rename the response GetDatabaseStatusResponseData and the containing DTO DatabaseStatusDTO (see GetISolateSuccessResponseData and IsolateDTO)